### ISSUE TITLE
two options 'show energy saving icon' and 'show night mode icon' (in 'energy saving options' and in 'night mode options'). features.

### ISSUE THREAD

**Issue by DI555** [AnySoftKeyboard/AnySoftKeyboard#2082]
i'd like to ask about features of showing 2 mini icons on a keyboard,-
it's two options 'show energy saving icon' 
and 'show night mode icon' !
In the option sections - 'energy saving options' and 'night mode options').
Because if those mini icones are displayed - they brake keuboard size, making it a little larger.
And even if that bug of brakings keyboard size bythose mini icons wiill besolved, many people and i don't want see that mini icons when 'energy saving' and 'night' modes are on !

**Comment by truenicoco** [comment]
I personally don't mind seeing the icons that much but this wasted space below the keyboard looks like an awful regression to me.

**Comment by DI555** [comment]
yes , the features to uncheck the shown icons imo should solve the problem with this wasted space below the keyboard!!!

 but even if this bug with large bottom border should  be solved - theese little icons was exactly on the enter key ;( and it's very uncomfortable , so really would be great to have options not to show them !!!
